js/src/contracts/registry.js

```diff
@@ -101,4 +101,17 @@ export default class Registry {
       return address;
     });
   }
+
+  lookupMeta (_name, key) {
```

Comment:
I see why... one calls `get` the other `getAddress` - so could be clever and obfuscate the code to pass the key as well as the relevant method to some helper or duplicate with a single-line difference for clarity. 

I opted for the latter.